src/pages/vedtak/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { erRevurderingIverksatt } from '~features/revurdering/revurderingUtils';
+import { Nullable } from '~lib/types';
+import { Behandling } from '~types/Behandling';
+import { IverksattRevurdering, Revurdering } from '~types/Revurdering';
+import { Sak } from '~types/Sak';
+import { Vedtak } from '~types/Vedtak';
+
+type Søknadsbehandlingsoppsummering = {
+    behandling: Behandling;
+    type: 'søknadsbehandling';
+};
+type Revurderingsoppsummering = {
+    revurdering: IverksattRevurdering;
+    forrigeBehandling: Behandling | Revurdering;
+    type: 'revurdering';
+};
```

Comment:
Bruk gjerne interface for sånne som dette

Reply:
sure!